client/src/ui/molecules/a11y-nav/index.tsx

```diff
@@ -46,6 +53,18 @@ export function A11yNav() {
           {"Skip to search"}
         </a>
       </li>
+      {showLangMenuSkiplink && (
+        <li>
+          <a
+            id="skip-select-language"
+            href="#select-language"
+            onClick={sendAccessMenuItemClick}
+            onContextMenu={sendAccessMenuItemClick}
+          >
+            {"Skip to select language"}
```

Comment:
```suggestion
            Skip to select language
```
No need to quote strings, same thing for the anchors

Reply:
Updated the other instances as well. Thanks!